components/proxy/src/test/java/com/hotels/styx/metrics/reporting/graphite/NonSanitizingGraphiteSenderTest.java

```diff
@@ -49,7 +49,7 @@ public void tearDown() {
         LocalNameServiceDescriptor.get().unset();
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
Did you mean to disable this test?

Reply:
Hi @dvlato. Well, no and yes. Do you still remember that DNS unit test class we implemented ages ago? I think that caused the test to start failing. We need to think if these tests are sensible to keep around.

In the meanwhile I'll check if we can re-enable this.

Reply:
Okay. This is now re-enabled. I had accidentally changed the Surefire plugin configuration, causing the failure.